pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/ClustersBase.java

```diff
@@ -522,8 +523,9 @@ private BrokerNamespaceIsolationData internalGetBrokerNsIsolationData(
             NamespaceIsolationPolicyImpl nsPolicyImpl = new NamespaceIsolationPolicyImpl(policyData);
             if (nsPolicyImpl.isPrimaryBroker(broker) || nsPolicyImpl.isSecondaryBroker(broker)) {
                 namespaceRegexes.addAll(policyData.getNamespaces());
-                brokerIsolationData.primary(nsPolicyImpl.isPrimaryBroker(broker));
-                brokerIsolationData.policyName(name);
+                if (nsPolicyImpl.isPrimaryBroker(broker)) {
+                    brokerIsolationData.primary(true);
+                }
```

Comment:
No need to change.

Reply:
Yes, It's an issue caused by rebase operation, I will fix it.